### PR TITLE
merge strings for train 47

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@mathjazz - incoming!

### CODE REVIEW

**Comment by rfk** [comment]
@shane-tomlinson please also use the scripting in this repo to send an email to dev-l10n-web for the new strings (and if it's not obvious how to do that, we've got a docs problem on our hands...)

**Comment by shane-tomlinson** [comment]
> @shane-tomlinson please also use the scripting in this repo to send an email to dev-l10n-web for the new strings (and if it's not obvious how to do that, we've got a docs problem on our hands...)

I will once the strings are merged. @mathjazz said he'd do it today (Oct 7, 2015).

**Comment by mathjazz** [comment]
I'll let you know as soon as I'm done, which will be later today when me and @zaach will be done with the TRIBE. ;) I had some problems connecting to VPN earlier (needed to SSH into Verbatim server), need to check what's going on.

**Comment by mathjazz** [comment]
r+

(I'm really sorry it took me so long to review this, will beer in Orlando be enough, @shane-tomlinson?

**Comment by shane-tomlinson** [comment]
@mathjazz - it looks like I have to rebase!

File: locale/af/LC_MESSAGES/server.po [hunk 1]
```diff
@@ -240,29 +240,206 @@ msgstr ""
...
+msgid ""
+"A device began syncing with the devices connected to your Firefox Account."
+msgstr ""
+
+#: templates/new_sync_device.html:5 templates/password_changed.html:5
+#: templates/password_reset.html:5 templates/post_verify.html:7
+#: templates/reset.html:7 templates/unlock.html:7
+#: templates/verification_reminder.html:7 templates/verify.html:7
+msgid ""
+"For more information, please visit <a href='%(supportUrl)s' style='color: "
```
Comment (zaach) [inline locale/af/LC_MESSAGES/server.po]: Didn't https://github.com/mozilla/fxa-auth-mailer/pull/92#issuecomment-145874949 get merged? Are we sticking with the old string?

Comment (shane-tomlinson) [inline locale/af/LC_MESSAGES/server.po]: The intent was to merge just after train 47 was cut and 48 started. Which git sha of the auth-mailer made train 47? If the sha includes the new stings, then I'll re-cut the strings.

**Comment by mathjazz** [comment]
NP @shane-tomlinson, feel free to merge.

**Pull request merged**